include/z64player.h

```diff
@@ -199,13 +199,13 @@ typedef enum PlayerMeleeWeaponAnimation {
 } PlayerMeleeWeaponAnimation;
 
 typedef enum PlayerDoorType {
-    /* -1 */ PLAYER_DOORTYPE_MINUS_1 = -1, // PLAYER_DOORTYPE_AJAR = -1,
+    /* -1 */ PLAYER_DOORTYPE_TALKING = -1, // Displays a message instead of opening
     /*  0 */ PLAYER_DOORTYPE_NONE,
     /*  1 */ PLAYER_DOORTYPE_HANDLE,
     /*  2 */ PLAYER_DOORTYPE_SLIDING,
-    /*  3 */ PLAYER_DOORTYPE_3, // PLAYER_DOORTYPE_FAKE // Unused?
+    /*  3 */ PLAYER_DOORTYPE_FAKE, // Unused? OoT leftover (?)
     /*  4 */ PLAYER_DOORTYPE_STAIRCASE,
-    /*  5 */ PLAYER_DOORTYPE_5
+    /*  5 */ PLAYER_DOORTYPE_PROXIMITY
```

Comment:
Given what was discussed earlier, this should probably get a similar name to the "force talk" actor flag.

Reply:
There's nothing on the code related to force talking for this flag. The milkbar door uses the `TALKING` doortype by default and after the player closes the text box it changes to `PROXIMITY`.
You can see on the video posted on the discord (https://discord.com/channels/688807550715560050/873211000122384415/1015980225072484363) that changing the doortype to `PROXIMITY` makes player open the door without having to press A and without any kind of text prompt.